arrow/locales.py

```diff
@@ -92,7 +146,11 @@ def describe(self, timeframe, delta=0, only_distance=False):
 
         return humanized
 
-    def describe_multi(self, timeframes, only_distance=False):
+    def describe_multi(
+        self,
+        timeframes: Collection[Tuple[TimeFrames, Union[int, float]]],
+        only_distance: bool = False,
+    ) -> str:
```

Comment:
I think we should restrict `timeframes` parameter to `Collection[Tuple[TimeFrames, Union[int, float]]]`.
Currently this method can accept arguments something like `[['seconds', 1, 1], (1, 1, 1)]`. Obviously no one use this method like that, but it would be good we can catch error on status analysis.

Reply:
I agree with the change--I don't think mixing tuples and lists in a container will see much usage.